client.go

```diff
@@ -100,6 +107,22 @@ func NewClient(endpoint, apiKey, apiSecret string) *Client {
 		client.TraceOn()
 	}
 
+	exoSecurityProvider, err := v2.NewSecurityProviderExoscaleV2(client.APIKey, client.apiSecret)
+	if err != nil {
+		panic(errors.Wrap(err, "unable to initialize security provider"))
```

Comment:
maybe panicking is a bit brutal here. Is there a way to return an explicit error without panicking ?

Reply:
Sadly no, because it would require us to change the `NewClient()` function signature to return an `error` which would break all implementations (we have to assume egoscale is used outside of our official tooling). If you think of a better way, feel free to suggest :)

Reply:
> Sadly no, because it would require us to change the NewClient() function signature to return an error which would break all implementations (we have to assume egoscale is used outside of our official tooling).

Indeed.